packages/eps/web-service/package.json

```diff
@@ -12,7 +12,7 @@
     "node": ">=14"
   },
   "scripts": {
-    "start": "node src/eps-web-service.js",
+    "start": "gulp build && node src/eps-web-service.js",
```

Comment:
Does the gulp need to be run on the development itteration cycle? The build could be attached to postinstall which actions on npm i. No big deal atm

Reply:
The quality gate is not passing though on test coverage. Maybe Brad can pick that up?